packages/turbo-repository/rust/src/internal.rs

```diff
@@ -79,7 +79,7 @@ impl PackageManagerRoot {
                     error: error.to_string(),
                     path: self.repo_state.root.clone(),
                 })?;
-        let package_manager = package_manager.clone();
+        let package_manager = *package_manager; // Clippy doesn't like the clone here, preferring deref instead.
```

Comment:
IMO `PackageManager` implementing `Copy` is clear enough reason to prefer `*` vs `clone`
```suggestion
        let package_manager = *package_manager;
```